webpack.config.js

```diff
@@ -24,8 +24,19 @@ module.exports = function(env = 'production') {
           'create-emotion',
           'polished',
           'prop-types',
-          '@leafygreen-ui/lib',
-          '@leafygreen-ui/theme',
+          ...[
```

Comment:
This is a list of all our current packages? 

This file is ran inside of a node context, meaning we have access to all node APIs.

As such we can make a function that reads all the directories inside of the packages/ directory and returns the list of their names, such that this array is generated off of the directory content and not something we have to manually sync. 

And even better...you could read all `package.json` files and take their `name` value such that there's no deriving of package names, we're directly referencing them.